### PR TITLE
Ev3 expansion set

### PR DESCRIPTION
**Pull request by dlech**

### CODE REVIEW

File: official_models/ev3/education_expansion/elephant/main.py [hunk 1]
```diff
@@ -38,20 +40,20 @@ def reset():
     neck_motor.run(750)
     while color_sensor.color() != Color.RED:
         wait(10)
-    neck_motor.stop(Stop.BRAKE)
+    neck_motor.hold()
```
Comment (laurensvalk) [inline official_models/ev3/education_expansion/elephant/main.py]: `brake()` ?

File: official_models/ev3/education_expansion/elephant/main.py [hunk 2]
```diff
@@ -38,20 +40,20 @@ def reset():
...
 
     # Run the trunk motor until the Touch Sensor is pressed.
     trunk_motor.run(600)
     while not touch_sensor.pressed():
         wait(10)
-    trunk_motor.stop(Stop.BRAKE)
+    trunk_motor.hold()
```
Comment (laurensvalk) [inline official_models/ev3/education_expansion/elephant/main.py]: `brake()` ?

File: official_models/ev3/education_expansion/elephant/main.py [hunk 3]
```diff
@@ -69,12 +71,12 @@ def grab():
...
     # Run a sequence of movements using the neck and trunk motors to
     # grab and pick up an object.
-    trunk_motor.run_angle(1000, 300, Stop.HOLD, False)
-    neck_motor.run_angle(1500, 350, Stop.BRAKE)
-    neck_motor.run_angle(-750, 350, Stop.BRAKE)
-    neck_motor.run_time(-150, 1000, Stop.HOLD, False)
-    trunk_motor.run_angle(-700, 500, Stop.BRAKE)
-    trunk_motor.run_angle(-300, 300, Stop.HOLD, False)
+    trunk_motor.run_angle(1000, 300, wait=False)
+    neck_motor.run_angle(1500, 350)
```
Comment (laurensvalk) [inline official_models/ev3/education_expansion/elephant/main.py]: Brake in several of these?

File: official_models/ev3/education_expansion/stair_climber/main.py [hunk 4]
```diff
@@ -59,29 +67,29 @@
...
-    brick.display.text(steps, (90, 70))
+    ev3.screen.clear()
+    ev3.screen.draw_text(70, 50, steps)
     wait(200)
 
     # Wait until any Brick Button is pressed.
-    while not any(brick.buttons()):
-        wait(10)
+    while not any(ev3.buttons.pressed()):
+        wait(100)
```
Comment (laurensvalk) [inline official_models/ev3/education_expansion/stair_climber/main.py]: Any reason to change to 100? Same for change from `pass` to 100 further above.

File: official_models/ev3/education_expansion/elephant/main.py (hunk 1 above)
Comment (dlech) [reply official_models/ev3/education_expansion/elephant/main.py]: We discussed removing `brake` altogether, so I removed all uses to see if we can get away with not actually using it or if it affects the models behavior too much.

File: official_models/ev3/education_expansion/stair_climber/main.py (hunk 4 above)
Comment (dlech) [reply official_models/ev3/education_expansion/stair_climber/main.py]: I should probably go back through and make them all consistent. I think different programs used different values.

**Comment by laurensvalk** [comment]
Updated imports, various motor fixes, workspaces.

**Pull request merged**